tests/foreman/ui/test_repository.py

```diff
@@ -761,6 +762,52 @@ def test_positive_sync_custom_repo_yum(self):
                     self.assertTrue(self.prd_sync_is_ok(repo_name))
 
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1487173)
+    @tier2
+    def test_positive_resync_custom_repo_after_invalid_update(self):
+        """Create Custom yum repo and sync it via the repos page. Then try to
+        change repo url to invalid one and re-sync that repository
+
+        :id: 089b1e41-2017-429a-9c3f-b0291007a78f
+
+        :expectedresults: Repository URL is not changed to invalid value and
+            resync procedure for specific yum repository is successful
+
+        :BZ: 1487173, 1262313
+
+        :CaseLevel: Integration
+        """
+        product = entities.Product(organization=self.session_org).create()
+        repo_name = gen_string('alpha')
+        with Session(self) as session:
+            self.products.search_and_click(product.name)
+            make_repository(
+                session,
+                name=repo_name,
+                url=FAKE_1_YUM_REPO,
+            )
+            self.assertIsNotNone(self.repository.search(repo_name))
+            self.setup_navigate_syncnow(
+                session,
+                product.name,
+                repo_name,
+            )
+            self.assertTrue(self.prd_sync_is_ok(repo_name))
+            self.repository.update(repo_name, new_url=INVALID_URL)
+            self.assertIsNotNone(self.user.wait_until_element(
+                common_locators['alert.error_sub_form']))
```

Comment:
In case BZ1487173 is not gonna be fixed in nearest future (which is possible since it's not top priority issue) - maybe instead of decorating entire test it's better to put this check under `if bz_bug_is_open` condition?